WebDriverAgentLib/Commands/FBSessionCommands.m

```diff
@@ -68,6 +68,9 @@ + (NSArray *)routes
     return FBResponseWithErrorFormat(@"'bundleId' desired capability not provided");
   }
   [FBConfiguration setShouldUseTestManagerForVisibilityDetection:[requirements[@"shouldUseTestManagerForVisibilityDetection"] boolValue]];
+  if ([requirements[@"maxTypingFrequency"] integerValue]) {
```

Comment:
probably we have just only test '[requirements[@"maxTypingFrequency"]' instead of '[requirements[@"maxTypingFrequency"] integerValue]'?